### PR TITLE
A11Y: Changes ported from ESAT

### PR DESCRIPTION
**Pull request by arvinsingla**
Porting various a11y changes from ESAT

### CODE REVIEW

File: modules/formulize/include/functions.php [hunk 1]
```diff
@@ -6817,15 +6817,12 @@ class='toggle-button-change-columns'
...
-				$html .= "
-					<div
-						class='elements-checkbox-list'
-						id='id-change-columns-toggle-panel-".$thisFid."'
-						aria-labelledby='id-change-columns-toggle-header-".$thisFid."'
-					>
-						".$boxeshtml."
-					</div>
-				";
+				$html .="<fieldset class='elements-checkbox-fieldset'>";
```
Comment (jegelstaff) [inline modules/formulize/include/functions.php:6820]: No 'aria-labelledby' in the new version of this, I assume that's on purpose? Just checking.

**Comment by jegelstaff** [comment]
Just one little question, but it looks good to me.

File: modules/formulize/include/functions.php (hunk 1 above)
Comment (arvinsingla) [reply modules/formulize/include/functions.php:6820]: Ahh! Good catch. The aria tag should probably be there.

Comment (arvinsingla) [reply modules/formulize/include/functions.php:6820]: Oh wait... I need to double check on this.

Comment (arvinsingla) [reply modules/formulize/include/functions.php:6820]: Ahh I see. This was the backport of accessible toggle work I believe.

**Pull request merged**